client/core/core.go

```diff
@@ -2261,6 +2279,12 @@ func (c *Core) createWalletOrToken(crypter encrypt.Crypter, walletPW []byte, for
 		return nil, err
 	}
 
+	if c.cfg.UnlockCoinsOnLogin {
+		if err = wallet.ReturnCoins(nil); err != nil {
+			c.log.Errorf("Failed to unlock all %s wallet coins: %v", unbip(wallet.AssetID), err)
+		}
+	}
+
```

Comment:
On wallet creation too?

Reply:
Correct.  It's in the docs and help for the cli switch.  You mean the field name should change?   `UnlockCoinsOnInitialAccess`?  The most routine use is login of course.